src/views/splash/splash.jsx

```diff
@@ -106,12 +114,33 @@ var Splash = injectIntl(React.createClass({
             if (!err) return this.setState({featuredGlobal: body});
         }.bind(this));
     },
-    getFeaturedCustom: function () {
+    getSharedByFollowing: function () {
         api({
-            uri: '/proxy/users/' + this.props.session.session.user.id + '/featured'
+            uri: '/projects/following/users',
+            authentication: this.props.session.session.user.token
```

Comment:
I'm having trouble figuring out what the value of this is, skimming through the code of this PR and around scratch-www (it looks like this is the first usage of the "authentication" `api` option), and I didn't get any clear results when testing on api-staging with header `X-token` and query `?x-token=` both equal to either the session token or the CSRF token. So I'm curious what the value of this would be?